app/models/mixins/relationship_mixin.rb

```diff
@@ -134,6 +134,16 @@ def parent_ids(*args)
     Relationship.resource_pairs(parent_rels(*args))
   end
 
+  # return a scope of all of the parents of this record
+  def parents_scope(*args)
+    options = args.extract_options!
+    if options[:of_type].nil? || options[:of_type].kind_of?(Array)
+      raise ArgumentError, ":of_type required as a single parameter"
+    end
```

Comment:
Not sure I like forcing this down the of_type path.  I know that's your current use case, but from a pure API perspective I would never expect that.